scrapy/extensions/httpcache.py

```diff
@@ -216,6 +220,8 @@ def __init__(self, settings):
         self.dbmodule = import_module(settings['HTTPCACHE_DBM_MODULE'])
         self.db = None
 
+        logger.debug("Using DBM cache storage in %(cachedir)s" % {'cachedir': self.cachedir})
```

Comment:
What do you think about logging exact filename of the database used for cache, not only a folder containing these databases? See open_spider method.